1.7/administration/installing/high-availability-setup.md

```diff
@@ -0,0 +1,89 @@
+
+# General Recommendations
+
+## Latency
+DC/OS master nodes should be connected to each other via highly available and low latency network links. This is required because some of the coordinating components running on these nodes use quorum writes for high availability. For example, Mesos masters, Marathon schedulers, and ZooKeeper use quorum writes.
+
+Similarly, most DC/OS services use ZooKeeper (or etcd, consul, etc) for scheduler leader election and state storage. For this to be effective, service schedulers should be connected to the ZooKeeper ensemble via a highly available, low-latency network link.
+
+## Routing
+DC/OS networking requires a unique address space. Cluster entities cannot share the same IP address. For example, apps and DC/OS agents must have unique address spaces.
+
+All IP addresses should be routable within the cluster.
+
+# Multi-AZ Recommendations
+Here are recommendations for specific multi-AZ configurations.
+
+**Important:** A typical DC/OS cluster has all master and agent nodes in the same zone. None of the following hybrid cloud and multi-DC setups have been explicitly tested or verified by Mesosphere.
+
+## Single-zone masters and cross-zone agents within a region
+All DC/OS masters are present in a single zone, but the agents can span multiple zones within that region.
+
+#### Recommendations
+
+- All agents within a zone should be tagged with an attribute (e.g., `zone:us-east-1a`) to easily constrain apps and services to specific zones.
+- Any DC/OS service schedulers should be scheduled to the same zone as masters (e.g., by using constraints).
+- For ease of failover of apps between zones, ensure that apps do not depend on zone level resources or that those resources are properly replicated and migrated on failover. Also ensure there is enough spare capacity in each zone to handle failover workload.
+
+#### Caveats
+
+- If an agent-only zone fails, all other zones will continue to function normally. Apps from the failed zone will be rescheduled into other zones as long as there is spare capacity.
+- If an agent-only zone gets partitioned from the masters-containing zone, apps will continue to run in that zone but schedulers might “reschedule” copies of the apps into other zones. 
+- If a masters-containing zone fails, apps in other zones continue to run. But app updates or rescheduling of failed apps is not possible, until a quorum of masters come up.
+- The total cross-sectional bandwidth between zones is limited. This means that performance can be degraded for data (I/O) intensive DC/OS services if deployed across zones.
+
+## Cross-zone masters and cross-zone agents within a region
+DC/OS masters and agents span multiple zones within a region.
+
+#### Recommendations
+
+- Quorum size and placement of masters should tolerate zone failure. For example, for a quorum size of 2, place each of the 3 masters in a separate zone.
```

Comment:
Changed to `Quorum size and placement of masters should tolerate zone failure. For example, for a quorum size of two, place each of the three masters in a separate zone. Do not place a quorum of masters into a single zone.`